HHS/Head-Start-TTADP
# [TTAHUB-2857] Prevent closing goals if they have open objectives

## Description of change
- Prevent closing goals if they have open objectives
- Add objective status change to the goal cards

## How to test
- Attempt to close a goal from the RTR when objectives are open
- Note that an error message appears. Verify the link goes to the correct recipient record page.
- Close those objectives from the RTR using the inline control
- Verify that suspending an objective triggers the modal
- Verify that once all the objectives are closed or suspended, you can close a goal from the RTR
- Verify that you can't change objective status if the goal is closed

## Issue(s)
* https://jira.acf.gov/browse/TTAHUB-2857

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
